roles/kubernetes/preinstall/tasks/0040-set_facts.yml

```diff
@@ -10,7 +10,7 @@
   set_fact:
     ansible_os_family: "RedHat"
     ansible_distribution_major_version: "8"
-  when: ansible_distribution in ["Kylin Linux Advanced Server", "openEuler"]
+  when: ansible_distribution in ["Kylin Linux Advanced Server", "openEuler", "UnionTech"]
```

Comment:
By setting `ansible_os_family: "RedHat"` here, I guess we don't need to change roles/kubernetes/preinstall/defaults/main.yml in this pull request for UnionTech.
Isn't it correct?

Reply:
https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/preinstall/tasks/0040-set_facts.yml#L174-L182

In this logic, if `ansible_os_family` is `UnionTech`, then the value of the `dhclienthookfile` variable will be `undefined`, which is why I want to set the variable `ansible_os_family` to `Redhat`.

Reply:
Maybe it could also be handled in the following way：
```
- name: target dhclient hook file for Red Hat family
  set_fact:
    dhclienthookfile: /etc/dhcp/dhclient.d/zdnsupdate.sh
  when: ansible_os_family == "RedHat" and ansible_os_family == "UnionTech"
```
@oomichi  What do you think？😊